governance.qmd

```diff
@@ -0,0 +1,96 @@
+
+If there is dissent by an administrator, they should post a reasoned reply within the one calendar month notice period. However, no one administrator has a veto. Where there is dissent, voting shall take place among administrators to decide on an issue, with a simple majority of votes required for a Proposal to pass. If a Proposal is passed, it will come into effect one calendar month after the Proposal was initially posted. If a Proposal is not passed within one calendar month of the posting of the Proposal, then the Proposal lapses and is not enacted until such a time that it passes.
+
+Voting shall take place publicly under the same Proposal that was initially posted.
+
+As illustration, if the total number of administrators is six, then at least four votes are required to pass a Proposal.
+
+## R-Multiverse Mission
+
+The mission of R-Multiverse is to provide a production-grade, open repository for the R community in general. R-Multiverse is committed to free and equal access, and its source code is released under the open-source MIT License.
+
+### R-Multiverse is non-commercial but commercially-aware
+
+We invite third parties, including for-profit corporations, to contribute to R-Multiverse and use it for their own purposes. However it is important for the longevity and community health of R-Multiverse that the community be recognised as the primary contributor to the project and that the project continues to serve all members of the community equally.
+
+The RMC actively track the corporate allegiances of administrators and moderators, and strives to ensure influence around any particular aspect of the project is not overly skewed towards a single corporate entity.
+
+### Reputation
+
+The RMC is responsible for protecting the R-Multiverse name, brand and reputation, and for enforcing adherence to this document.
+
+Where this is jeopardised through the actions of an administrator, moderator or contributor, the RMC will have the authority to immediately remove that person from their capacity in that role, not requiring the usual one month notice period for enacting Proposals.
+
+## Pre-Launch Period
+
+The R-Multiverse project is currently in what is considered a 'pre-launch' period, where the infrastructure is being actively developed, and policies are required to change to accommodate. It is beneficial in this period to adopt more flexible decision-making, and hence, during this period only, all mentions of 'one calendar month' in this document shall be read as 'two calendar weeks'.
+
+It is expected that this section of the document will be removed at a later date.
+
+## Project Leadership
+
+### R-Multiverse Management Committee (RMC)
+
+Name in alphabetical order (affiliation)
+
+- Charlie Gao (Hibiki AI)
+- Will Landau (Eli Lilly & Co.)
+- Jeroen Ooms (rOpenSci)
+- Maëlle Salmon (rOpensci)
```

Comment:
should this track more of my allegiances :grin:

Reply:
Yes, will add Cynkra. Any others I should be aware of? I guess rOpenSci was top of mind here, as we already have two members and if there were another then the entire project could be 'captured'. This is part of what the 'tracking affiliations' section aims to mitigate against.